src/components/WorkOrder/EditWorkOrder.tsx

```diff
@@ -61,8 +59,8 @@ const EditWorkOrderDescription = ({
     const postNoteResponse = await postNote(noteData)
 
     if (!editDescriptionResponse.success)
-      throw new Error('Failed to edit description')
-    if (!postNoteResponse.success) throw new Error('Failed to post note')
+      setError(editDescriptionResponse.error)
+    if (!postNoteResponse.success) setError(postNoteResponse.error)
```

Comment:
If both fail, the first error is being overwritten. Maybe check if the first request fails before making the second request